README_validations.md

```diff
@@ -0,0 +1,106 @@
+oscal.xml validates
+```
+
+There is no detailed output, just a statement that indicates validation has been performed successfully.
+
+If you remove the `--noout` flag from the command line, a copy of the input will be returned. This is because the utility is designed for a pipeline architecture. Its outputs will be provided as inputs to the next tool in the pipeline.
+
+As for OSCAL JSON, the results from validation with AJV will look like this:
+
+```
+# Install the ajv-formats library in NodeJS runtime to avoid runtime errors.
+npm install -g ajv-formats ajv-cli
+ajv-cli validate -s oscal-schema.json -d oscal.json -c ajv-formats
+oscal.json valid
+```
+
+## What does an error look like?
+
+### Well-formed, but not valid
+If your document is not valid, the results may look like this:
+
+```
+Element '{http://csrc.nist.gov/ns/oscal/1.0}last-modified': This element is not expected. Expected is ( {http://csrc.nist.gov/ns/oscal/1.0}title ).
+test-invalid-profile.xml fails to validate
+```
+
+The important line here is the final one. However, there may be one or more messages preceding it, describing the finding of the parser that a validation rule has been violated.
+
+### Not even well-formed
+
+It can happen, however, that the file is not even legible to the processor ("parser"), because it is not well-formed to the syntax rules.
+
+In such a case, XML Lint signals trouble like this:
+
+```
+test-invalid-profile.xml:10: parser error : Opening and ending tag mismatch: profile line 2 and metadata
+  </metadata>
+             ^
+test-invalid-profile.xml:11: parser error : Extra content at the end of the document
+  <merge>
+  ^
+```
+
+In this case the processor reports the syntax issues it detects, to the extent possible, for a document that is not well-formed. The processor will be unable to complete validation of the document because it is not well-formed.
+
+## What can you learn from errors?
+
+A validation error tells you first and foremost, that your file is incorrect in a basic way. For example, an element (data structure) may be out of place, or missing a required piece. Tools that improperly implement schema validation, misconfigured tools, data quality issues, workflow issues, or many other causes may be the cause. They also vary in how easy they are to correct.
+
+The most important consideration in correcting a validation error is selecting the appropriate schema and the appropriate version of that schema. OSCAL maintains backward compatibility across all schemas as expressed through [the use of semantic versioning](https://github.com/usnistgov/OSCAL/blob/main/docs/content/downloads/_index.md#oscal-releases).
+
+A *well-formedness error*, however, tells you your file is incorrect in a more fundamental way. It is both more severe (further processing cannot continue) but also easier to fix. In a mature system they should be rare, and when they do occur it is because preliminary checks for well-formedness were not implemented or ignored.
```

Comment:
```suggestion
A *well-formedness error*, however, tells you your file is incorrect in a more fundamental way. It is both more severe (further processing cannot continue) but also easier to address by using tools correctly. In a mature system, well-formedness errors should be rare, and when they do occur it is because preliminary checks for well-formedness were not implemented or ignored.
```